tests/lib/kubernetes/kubernetes_base.py

```diff
@@ -135,6 +135,13 @@ def get_pod_by_app_label(self, label, namespace="rook-ceph"):
             % (namespace, label)
         )[1].strip()
 
+    def get_service_by_app_label(self, label, namespace="rook-ceph"):
```

Comment:
To disambiguate should we call this `get_services_by_app_label`? Perhaps we should also then split \n and return a list of services.

( the get_pod_by_app_label should probably also be improved like this separately)

Reply:
Changed both.